src/cogent3/app/composable.py

```diff
@@ -887,14 +883,12 @@ def _add(self, other):
 def _repr(self):
     val = f"{self.input!r} + " if self.app_type is not LOADER and self.input else ""
     all_args = deepcopy(self._init_vals)
-    args_items = all_args.get("args")
+    args_items = all_args.pop("args", None)
```

Comment:
@GavinHuttley 
I replaced combination of `get` and `pop` with only one `pop` command.